src/main/webapp/scripts/jquery.infiniteScroller.js

```diff
@@ -0,0 +1,40 @@
+/* 
+ * Copyright 2014 Jasig.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+(function ( $ ) {
+ 
+ 
+    $.fn.infiniteScroll = function(options) {
+        
+        var settings = $.extend({
+            // These are the defaults.
+            load: function() {
+                return "<h1>LOADED</h1>";
```

Comment:
Looks like code that runs only if you're not doing it properly.  Maybe a helpful message?